packages/cli/CHANGELOG.md

```diff
@@ -5,3 +5,4 @@ All notable changes to the Kubernetes Secrets Credential Manager Plug-in for Zow
 ## Recent Changes
 
 - Added system test for v2 profiles
+- Added readme with instructions on how to use and install the repo
```

Comment:
Please add a link to the related issue or PR so that we can included that in the release notes